extension/src/config/config.js

```diff
@@ -0,0 +1,55 @@
+const _ = require('lodash')
+
+const configPath = process.env.CONFIG_PATH
+
+function getEnvConfig () {
+  return {
+    port: process.env.PORT,
+    logLevel: process.env.LOG_LEVEL,
+    apiExtensionBaseUrl: process.env.API_EXTENSION_BASE_URL
+  }
+}
+
+function getCTPEnvCredentials () {
+  return {
+    projectKey: process.env.CTP_PROJECT_KEY,
+    clientId: process.env.CTP_CLIENT_ID,
+    clientSecret: process.env.CTP_CLIENT_SECRET,
+    apiUrl: process.env.CTP_API_URL,
+    authUrl: process.env.CTP_AUTH_URL
+  }
+}
+
+function getFileConfig () {
+  let fileConfig = {}
+  try {
+    fileConfig = require(configPath) // eslint-disable-line
+  } catch (e) {
+    // config file was not provided
```

Comment:
log it?

Reply:
This method is removed: https://github.com/commercetools/commercetools-adyen-integration/pull/4/commits/0f21fef327bad8c18c558665e226e79dd5f01816